lsp-mode.el

```diff
@@ -4056,7 +4063,9 @@ LSP server result."
                            (-when-let ((&SnippetTextEdit :range (&RangeToPoint :start)
                                                          :insert-text-format? :new-text) edit)
                              (when (eq insert-text-format? lsp/insert-text-format-snippet)
-                               (lsp--expand-snippet new-text start (+ start (length new-text)))))))))
+                               (lsp--expand-snippet new-text start (+ start (length new-text))))))
+                         (when lsp-auto-save-changed-buffers
```

Comment:
This apply for every edits in a file, which can be expensive, can you it just one when the apply text edits finished?